extensions/libraries/redcore/api/api.php

```diff
@@ -261,10 +261,12 @@ public static function getPostedData()
 			$inputData = $input->post->getArray();
 		}
 
-		// Filter data with JInput default filter
-		$postedData = new JInput($inputData);
+		$filter = JFilterInput::getInstance(array(), array(), 1, 1, 1, -1);
```

Comment:
I'm not sure what you mean?

JFilterInput::getInstance
```php
public static function &getInstance($tagsArray = array(), $attrArray = array(), $tagsMethod = 0, $attrMethod = 0, $xssAuto = 1, $stripUSC = -1)
```

Do you want me to leave the last parameter out?